packages/functional-tests/lib/query-params.ts

```diff
@@ -29,5 +29,6 @@ export const syncDesktopV3QueryParams = new URLSearchParams({
   service: 'sync',
   action: 'email',
   automatedBrowser: 'true',
-  forceUA: uaStrings['desktop_firefox_79'],
+  // TODO: Should we add a new string? This was desktop_firefox_79, but that's not a listed string...
```

Comment:
What do you mean by "listed string"?

I don't know what the relevance of 71 vs 79 is but my guess is we just used whatever was latest when we wrote the original test, so probably fine 🤷‍♀️

Reply:
By listed I just meant defined. Firefox_72 isn't the list of defined uaStrings.

Reply:
Ah I see. If it passes, I imagine it's fine, @vbudhram didn't seem concerned so :shipit: